bindings_test.go

```diff
@@ -968,14 +926,27 @@ func TestFilterPredicates(t *testing.T) {
 
 		q, _ := NewQuery([]byte(testCase.query), getTestGrammar())
 		qc := NewQueryCursor()
-		qc.Exec(q, root)
+		qc.Exec(q, root, []byte(testCase.input))
 
-		before, ok := qc.NextMatch()
+		qm, ok := qc.nextMatch(false)
```

Comment:
It's mostly for unit testing, otherwise getting a `QueryMatch` that has `satisfiesTextPredicates() == false` to test that function would be annoying to construct.